addons/pos_sale/static/tests/tours/PosSaleTour.js

```diff
@@ -199,6 +199,13 @@ registry.category("web_tour.tours").add("PosQuotationSaving", {
             ProductScreen.selectedOrderlineHas("Product", "4.00", "40.00"),
             ProductScreen.controlButton("More..."),
             ProductScreen.controlButton("Park Order"),
+            {
+                trigger: ".pos",
+                run: () => {
+                    window.location.reload();
+                },
+            },
+            ProductScreen.orderIsEmpty(),
```

Comment:
Can you maybe include these steps directly in POS ? Not in a submodule.

Thanks

Reply:
It's the only test that already uses the Park Order feature. Should I modify another test from the PoS modules to have it there ?